tbx/people/tests/test_author.py

```diff
@@ -0,0 +1,122 @@
+from django.test import TestCase
+
+from tbx.core.factories import HomePageFactory
+from tbx.images.factories import CustomImageFactory
+from tbx.people.models import Author, PersonIndexPage, PersonPage
+from tbx.taxonomy.factories import TeamFactory
+from wagtail.models import Site
+
+
+class TestAuthor(TestCase):
+    def setUp(self):
+        super().setUp()
+
+        site = Site.objects.get(is_default_site=True)
+        root = site.root_page.specific
+        home = HomePageFactory(parent=root)
+
+        site.root_page = home
```

Comment:
**Comment** (nitpick): It's a little odd, but not at all disallowed, to have a site pointing at a non-top level page. I'm sure it's inconsequential, but you can add a home page at the top level (well, one below the treebeard root page) with:

```suggestion
        site = Site.objects.get(is_default_site=True)
        root = Page.objects.get(depth=1)  # or .get(path="0001")
        home = HomePageFactory(parent=root)

        site.root_page = home
```